### PR TITLE
add honduras_sefin_bulk spider

### LINKED ISSUE #913: Honduras SEFIN Bulk download
**Links to bulk download and/or API documentation**
https://www.sefin.gob.hn/edca/

**Links to bulk download and/or API endpoints**
Bulk download https://www.sefin.gob.hn/edca/
API https://guancasco.sefin.gob.hn/EDCA_WEBAPI/swagger/ui/index (the API endpoints return 500)

**Priority**
anytime

**Data structure**
Zip files with a JSON Line file with one release package per line

**Publication format**
Zip files with a JSON Line file with one release package per line

### PR DESCRIPTION
**Pull request by Ravf95**
Closes #913

### CODE REVIEW

**Review by yolile — CHANGES_REQUESTED** [review]

File: kingfisher_scrapy/spiders/honduras_sefin_bulk.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+from kingfisher_scrapy.util import components
+
+
+class HondurasSEFINBulk(CompressedFileSpider, PeriodicSpider):
+    """
+    Domain
+      Secretaria de Finanzas de Honduras (SEFIN)
+    Spider arguments
+      from_date
+        Download only data from this year onward (YYYY format).
```
Comment (yolile) [inline kingfisher_scrapy/spiders/honduras_sefin_bulk.py:11]:
```suggestion
        Download only data from this year onward (YYYY format). Defaults to '2012'.
```

File: kingfisher_scrapy/spiders/honduras_sefin_bulk.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        If ``until_date`` is provided, defaults to '2012'.
```
Comment (yolile) [inline kingfisher_scrapy/spiders/honduras_sefin_bulk.py:12]:
```suggestion
```

File: kingfisher_scrapy/spiders/honduras_sefin_bulk.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      until_date
+        Download only data until this year (YYYY format).
```
Comment (yolile) [inline kingfisher_scrapy/spiders/honduras_sefin_bulk.py:14]:
```suggestion
        Download only data until this year (YYYY format). Defaults to the current year.
```

File: kingfisher_scrapy/spiders/honduras_sefin_bulk.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        If ``from_date`` is provided, defaults to the current year.
```
Comment (yolile) [inline kingfisher_scrapy/spiders/honduras_sefin_bulk.py:15]:
```suggestion
```

Comment (Ravf95) [reply kingfisher_scrapy/spiders/honduras_sefin_bulk.py:15]: Thanks, I'll apply all suggestions to the next commit

**Review by yolile — APPROVED** [review]

**Pull request merged**